### ISSUE TITLE
Implement batch read/write into scdssdhc2

### ISSUE THREAD

**Issue by lifehackerhansol** [DS-Homebrew/DLDI#4]
This is missing in all source codes for SCDSSDHC/DSTT.

In LogicCardWrite https://github.com/DS-Homebrew/DLDI/blob/master/scdssdhc2/source/iointerface.c

```c
	command[7] = 0x51;
	command[6] = (address >> 24) & 0xff;
	command[5] = (address >> 16) & 0xff;
	command[4] = (address >> 8)  & 0xff;
	command[3] =  address        & 0xff;
	command[2] = 25; // as opposed to 24 in single write
	command[1] = 1;
	command[0] = 0;
```
can be done to start a batch write.

If the written sector is the last sector, then between 0x50 and 0x56 commands, we need to send this command
```c
	command[7] = 0x51;
	command[6] = 0;
	command[5] = 0;
	command[4] = 0;
	command[3] = 0;
	command[2] = 12; // signal that that was the last block
	command[1] = 1;
	command[0] = 0;
```
and then loop.

The original R4 SD framework wasn't really designed for batch writes, as the original R4 doesn't have that implemented. So a rewrite of LogicCardWrite may be necessary.

There is no batch reads in the DSTT, however it's fast enough that this doesn't exactly matter.

**Comment by lifehackerhansol** [comment]
It appears batch **reads** are also possible.

Instead of 0x53 before cardPolledTransfer, we can do 0x54 once, and after each chunk read we send 
```c
command[7] = 0x51;
command[1] = 0x7;
```
to tell the card to read the next chunk. When we don't send the above command, it seems to signal the end of read.

Write has been implemented in f79fdc4b1dab45e5fe8757a05851a77a98ce7bbc.

**Comment by lifehackerhansol** [comment]
Batch read also requires 
```c
command[7] = 0x51;
command[2] = 12;
command[1] = 1;
```
sent, with the usual wait-and-get-reply commands.
